### DISCUSSION TITLE
EOLing Ruby 2.6 and 2.7 in Dalli 4.0

### DISCUSSION THREAD

**Discussion by petergoldstein** [petergoldstein/dalli#950] [Announcements]
Ruby 2.7 hits EOL on March 31, 2023.  Shortly thereafter I'm planning to release Dalli 4.0, which will require Ruby 3.0+.

There are a few outstanding issues and PRs which I'll attempt to work through over the next couple of months, with the intention of getting them out into a 3.2.x version before releasing Dalli 4.0.

If there are any questions or concerns about this approach, please let me know.  Thanks!

**Comment by jamiemccarthy** [comment]
Sounds good to me!
